docs/source/general/setup.rst

```diff
@@ -32,7 +32,7 @@ If you want to contribute, install your cloned fork:
 
     git clone git@github.com:<your_fork>/django-freeradius.git
     cd django-freeradius
-    python setup.py develop
+    sudo python setup.py develop
```

Comment:
this shows you are not using a virtualenv, we assume everyone installing the application is using a virtualenv which does not require sudo, maybe we should explain that instead of adding sudo here

Reply:
Ok, I will use virtualenv (just I have a lot of VMs special for this task). I will try to documentate virtualenv.

Reply:
Great